js/ui/list/ui.list.edit.decorator.reorder.js

```diff
@@ -1,33 +1,33 @@
-var $ = require('../../core/renderer'),
-    eventsEngine = require('../../events/core/events_engine'),
-    extend = require('../../core/utils/extend').extend,
-    eventUtils = require('../../events/utils'),
-    registerDecorator = require('./ui.list.edit.decorator_registry').register,
-    EditDecorator = require('./ui.list.edit.decorator'),
-    Sortable = require('../sortable');
-
-var REORDER_HANDLE_CONTAINER_CLASS = 'dx-list-reorder-handle-container',
-    REORDER_HANDLE_CLASS = 'dx-list-reorder-handle',
-    REOREDERING_ITEM_GHOST_CLASS = 'dx-list-item-ghost-reordering',
-    STATE_HOVER_CLASS = 'dx-state-hover';
+import $ from '../../core/renderer';
+import eventsEngine from '../../events/core/events_engine';
+import { extend } from '../../core/utils/extend';
+import { isMouseEvent } from '../../events/utils';
+import { register as registerDecorator } from './ui.list.edit.decorator_registry';
+import EditDecorator from './ui.list.edit.decorator';
+import Sortable from '../sortable';
+
+const REORDER_HANDLE_CONTAINER_CLASS = 'dx-list-reorder-handle-container';
+const REORDER_HANDLE_CLASS = 'dx-list-reorder-handle';
+const REORDERING_ITEM_GHOST_CLASS = 'dx-list-item-ghost-reordering';
+const STATE_HOVER_CLASS = 'dx-state-hover';
 
 registerDecorator(
     'reorder',
     'default',
     EditDecorator.inherit({
 
         _init: function() {
-            let list = this._list;
+            const list = this._list;
 
             this._groupedEnabled = this._list.option('grouped');
 
             this._lockedDrag = false;
 
-            this._sortable = list._createComponent(list._scrollView.$content(), Sortable, extend({
+            this._sortable = list._createComponent(list._scrollView.content(), Sortable, extend({
                 component: list,
                 contentTemplate: null,
                 allowReordering: false,
-                filter: '.dx-list-item',
+                filter: '> .dx-list-item, > .dx-list-group > .dx-list-group-body > .dx-list-item',
```

Comment:
Why not use this._groupedEnabled variable to choose correct filter instead of sortable modification?

`filter: this._groupedEnabled? '> .dx-list-item': '> .dx-list-group> .dx-list-group-body> .dx-list-item'`

Reply:
Done